### PR TITLE
[Feature] Update pytorch2onnx

### PR DESCRIPTION
**Pull request by grimoire**
Hi,
This PR adds restorer model support to pytorch2onnx.
Dynamic shape, dynamic batch and visualize support are also added too.

### CODE REVIEW

File: tools/pytorch2onnx.py [hunk 1]
```diff
@@ -12,12 +13,28 @@
 from mmedit.models import build_model
 
 
+def show_result_pyplot(img, title='', block=True):
```
Comment (innerlee) [inline tools/pytorch2onnx.py:16]: So this function simply shows an image? Can you rewrite it using cv2.imshow() or imwrite?

**Review by innerlee — APPROVED** [review]

File: tools/pytorch2onnx.py [hunk 2]
```diff
@@ -32,85 +35,144 @@ def pytorch2onnx(model,
...
     model.forward = model.forward_dummy
     # pytorch has some bug in pytorch1.3, we have to fix it
     # by replacing these existing op
     register_extra_symbolics(opset_version)
+    dynamic_axes = None
+    if dynamic_export:
+        dynamic_axes = {
+            'input': {
+                0: 'batch',
+                2: 'height',
```
Comment (RunningLeon) [inline tools/pytorch2onnx.py:54]: Do we support dynamic shape feature for ESRGAN and SRCNN in this tool?

File: tools/pytorch2onnx.py [hunk 3]
```diff
@@ -32,85 +35,144 @@ def pytorch2onnx(model,
...
+                3: 'width'
+            },
+            'output': {
+                1: 'batch',
```
Comment (RunningLeon) [inline tools/pytorch2onnx.py:58]: Index of batch for output tensor is `1` or `0` ?

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**